README.md

```diff
@@ -13,6 +13,8 @@ iex(1)> Extrace.calls([{Enum, :take_random, fn _ -> :return end}, {Enum, :count,
 4
 ```
 
+Note that the functions to be traced (`:take_random` and `:count` in the example above) cannot be private.
```

Comment:
the function could be private if `scope: :local` is set

Reply:
OK, thanks.

FYI @mccauli